Albums/Albums/ViewControllers/AlbumsTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AlbumsTableViewController.swift
+//  Albums
+//
+//  Created by Chris Gonzales on 3/9/20.
+//  Copyright © 2020 Chris Gonzales. All rights reserved.
+//
+
+import UIKit
+
+class AlbumsTableViewController: UITableViewController {
+    
+    // MARK: - Properties
+    
+    var albumController: AlbumController?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let albumController = albumController else { return }
+        albumController.getAlbums { (error) in
+            if let error = error{
+                NSLog("Error getting albums: \(error)")
+                return
+            }
+            self.tableView.reloadData()
```

Comment:
Would want to call this on the main queue since getAlbums is done on a background thread.